packages/jest-message-util/src/types.ts

```diff
@@ -1,12 +1,5 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
```

Comment:
Oops I guess this is supposed to exist, though I don't know what the point is. I don't even think you could copyright this... Maybe there needs to be a lint rule making sure that these comments exist where they are supposed to?